extensions/amp-google-assistant-assistjs/amp-google-assistant-assistjs.md

```diff
@@ -0,0 +1,2 @@
+# `amp-google-assistant-assistjs`
+This extension brings some of the Assistant capabilities to the AMP pages that includes this extension and its custom elements.
```

Comment:
Can you elaborate on this a little bit. I don't really understand from this what the extension will do, thanks.

If you plan to flush this out in a later commit, maybe add a TODO indicating as such.

Reply:
Done